internals/cli/cmd_enter.go

```diff
@@ -139,6 +141,18 @@ func (cmd *cmdEnter) Execute(args []string) error {
 		panic("internal error: expected subcommand (parser.Active == nil)")
 	}
 
+	// The exec command makes the terminal raw to run interactive
+	// applications. Ignore --verbose for those cases, to avoid
+	// log output being sent to the interactive terminal.
+	if execCmd, ok := commander.(*cmdExec); ok {
+		stdinIsTerminal := ptyutil.IsTerminal(unix.Stdin)
+		stdoutIsTerminal := ptyutil.IsTerminal(unix.Stdout)
+		if stdinIsTerminal && execCmd.useTerminal(stdoutIsTerminal) {
```

Comment:
Sorry to be nit-picky, but let's change this `if` statement so it has the same "shape" as the `if` statement in `cmd_exec.go` -- that way it's easy to see they're identical.

```suggestion
		terminal := execCmd.useTerminal(stdoutIsTerminal)
		if terminal && stdinIsTerminal {
```